src/contrib/mlir/core/pass/ng_dialect_builder.cpp

```diff
@@ -461,6 +590,10 @@ mlir::Operation* NgDialectConversionPass::createGenericOp(const ngraph::Node* ng
 
         auto argV = getTensorValue(argTensor.get()).m_value;
         argValues.push_back(argV);
+        if (as_type<const ngraph::op::Softmax>(ngNode))
```

Comment:
I really don't like this. Can we instead pass an argument with the # of args we want to convert ?

Reply:
If you're not using the value, use `is_type` which just checks the type and doesn't do the cast.